model/Hardware/Classes/CreationProcess.md

```diff
@@ -0,0 +1,27 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# CreationProcess
+
+## Summary
+
+The Creation Process refers to the abstract process class used to produce products.
+
+## Description
+
+The creation process refers to the systematic steps involved in bringing something new into existence. This can apply to products, ideas, businesses, art, software, and even life itself. 
+Relationship: 
+For Each `CreationProcess` there is at least one `/Core/Relationship` class or subclass with the relationshipType of 'hasOutput’ on the from and an `/Core/Element` class or subclass on the to. 
```

Comment:
```suggestion

Relationship:

For every `CreationProcess`, there exists at least one `/Core/Relationship`
class or subclass with a `relationshipType` of 'hasOutput',
connecting the `CreationProcess` (from)
to a `/Core/Element` class or subclass (to).
```

"on the from" is quite disconnecting from `CreationProcess`.

Will the suggested paragraph make it easier to read and still convey the same meaning?